index.html

```diff
@@ -243,5 +243,44 @@ <h2>
       </span>
       <hr class="solid" />
     </section>
+    <footer class="form-container">
+      <section class="contact-form">
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/mrgjlwwj" method="POST">
+          <ul>
+            <li>
+              <input
+                type="text"
+                name="name"
+                value="Yeremias"
+                required
+                maxlength="30"
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                name="email"
+                value="natajayanj14@gmail.com"
+                required
+              />
+            </li>
```

Comment:
- [ ] While you have done great work, please add a placeholder to each for your input field, actually your design looks like this
<img width="440" alt="plsce" src="https://user-images.githubusercontent.com/79058364/159681841-8781df4a-d571-4dff-9c30-9c0ff44ca213.png">

- You can do something like 
    - for the name field: placeholder="Full name"
    - for the email field: placeholder="Email"